src/Workspaces/VisualBasic/Portable/Extensions/ContextQuery/SyntaxTreeExtensions.vb

```diff
@@ -1063,8 +1063,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions.ContextQuery
             tokenOnLeftOfPosition = tokenOnLeftOfPosition.GetPreviousTokenIfTouchingWord(position)
 
             If tokenOnLeftOfPosition.IsKind(SyntaxKind.OpenParenToken) Then
-                Return tokenOnLeftOfPosition.Parent.IsKind(SyntaxKind.ParenthesizedExpression,
-                                                           SyntaxKind.TupleExpression, SyntaxKind.TupleType)
+                ' If(bar$$
+                ' Exclude parenthesized expressions inside if statements
+                If (tokenOnLeftOfPosition.Parent.IsKind(SyntaxKind.ParenthesizedExpression)) Then
+                    Return Not tokenOnLeftOfPosition.Parent.Parent.IsKind(SyntaxKind.IfStatement)
```

Comment:
Why does this change actually work? This is entirely valid (if a bit silly) code:

    If (x, x).Equals((0, 0)) Then

Does this impact that?

Reply:
Also, what about other constructs like Do While?